### PR TITLE
Adding prescale index and L1 tables with per-type data

### PR DESCRIPTION
**Pull request by avetisya**
This adds the prescale index (as seen by the CMSSW framework) and L1 tables split by Physics, Calibration and Random. The prescale index is currently passes the "sample" operation to hltd and the transfer system so the latter will take it from a random file. The functionality to compare across all files and verify that it is the same is not currently present, although this is done in the CMSSW code so at least if it changes for the same process, we will know.

There were also a few changes to clean up the code and remove some ambiguities. The iSummary variables are now explicitly initialized at the beginning of each lumi section. Also, L1Accept is now L1AlgoAccept to clarify that this is for the algorithm bits only rather than all bits.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

File: HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc [hunk 1]
```diff
@@ -26,10 +26,10 @@ TriggerJSONMonitoring::TriggerJSONMonitoring(const edm::ParameterSet& ps)
 
   triggerResultsToken_ = consumes<edm::TriggerResults>(triggerResults_);
 
-  if (ps.exists("L1Results")) level1Results_ = ps.getParameter<edm::InputTag>("L1Results");  //DS 
-  else                        level1Results_ = edm::InputTag("hltGtDigis");                  //DS
+  if (ps.exists("L1Results")) level1Results_ = ps.getParameter<edm::InputTag>("L1Results");
```
Comment (Martin-Grunewald) [inline HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc]: Putting the proper value into the fillDescriptions(), the ps.exists is not needed!

**Comment by avetisya** [comment]
Hi Martin,

I've removed the ps.exists conditionals. The proper values were already in fillDescriptions().

-- Aram

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Martin-Grunewald** [comment]
Hi Aram @avetisya 
Please make a PR for 75X as well, as 74X PRs are no longer forward ported as of today (ORP meeting).

**Comment by avetisya** [comment]
Hi Martin,

The 75X version is #9441 .

-- Aram

**Comment by Martin-Grunewald** [comment]
@avetisya 
Thanks!

**Comment by davidlange6** [comment]
+1

**Pull request merged**